uaclient/messages/__init__.py

```diff
@@ -1850,6 +1850,10 @@ def __repr__(self):
     "unattended-upgrades-cfg-value-turned-off",
     t.gettext("{cfg_name} is turned off"),
 )
+UNATTENDED_UPGRADES_UNINSTALLED = NamedMessage(
+    "unattended-upgrades-uninstalled",
+    "unattended-upgrades packages is not installed",
+)
```

Comment:
Sorry I missed this initially
```suggestion
UNATTENDED_UPGRADES_UNINSTALLED = NamedMessage(
    "unattended-upgrades-uninstalled",
    t.gettext("unattended-upgrades packages is not installed"),
)
```

Reply:
Good catch